src/renderer/vanilla/util/render.js

```diff
@@ -10,6 +10,28 @@ const convertStringToHTML = (templateString) => {
 	return document.querySelector('main');
 };
 
+const isTemplateElement = (element) => {
+	const tagName = element.node?.tagName?.toLowerCase() ?? false;
+	if (!tagName || !tagName.includes('-')) {
+		return false;
+	}
+
+	if (_cachedTemplateElements[tagName]) {
+		return true;
+	}
+
+	const elementClass = customElements.get(tagName);
```

Comment:
```suggestion
	const elementClass = window.customElements.get(tagName);
```
Can we always add the `window.`before global variables? To make it more safe and more explicit.
Does this also run on the server? If so it won't work right?

Reply:
No this won't run on the server, but we can add it yes. This part of code needs to be changed anyway, as the check is not correct. But i would do this in a different pull request. I just moved it, but i will reset it to it's original position

Reply:
👍